atd-vze/src/views/Users/Users.js

```diff
@@ -11,15 +11,62 @@ import {
   Pagination,
   PaginationItem,
   PaginationLink,
+  Popover,
   Row,
   Table,
   Button,
   Spinner,
+  PopoverBody,
 } from "reactstrap";
 import Can from "../../auth/Can";
 import { useAuth0 } from "../../auth/authContext";
 import { rules } from "../../auth/rbac-rules";
 
+// Call the list_users API and return the users for the current page
+async function getCurrentPageUsers(page, perPage, token) {
+  const endpoint = `${process.env.REACT_APP_CR3_API_DOMAIN}/user/list_users?page=${page}&per_page=${perPage}`;
+  const res = await axios.get(endpoint, {
+    headers: {
+      Authorization: `Bearer ${token}`,
+    },
+  });
+  return res;
+}
+
+// Fetch an array of all the users
+async function getAllUsers(pageCount, perPage, totalUsers, setAllUsers, token) {
+  let page = 0;
+  let users = [];
+  // query every page of users until we have fetched them all
+  while (page <= pageCount) {
+    const res = await getCurrentPageUsers(page, perPage, token);
+    res.data.users.forEach(user => {
+      // make sure the user is not blocked/inactive, then push to an array
+      if (user.status === false || user.status === undefined) {
```

Comment:
just to double-check: `status === false || status ==== undefined` means the user is active?

Reply:
Yeah it sounds counterintuitive, but this definitely seems to be working as expected in terms of copying all 107 active users to clipboard. I just tested some things and changed this to an if else statement to capture users that would fall under "blocked/inactive" and we dont have any. Fwiw i did not write this part of the code